scriptmodules/emulators/mupen64plus/mupen64plus.sh

```diff
@@ -292,33 +282,23 @@ function testCompatibility() {
 
     # fix Audio-SDL crackle
     iniConfig " = " "\"" "$config"
-    # create section if necessary
-    if ! grep -q "\[Audio-SDL\]" "$config"; then
-        echo "[Audio-SDL]" >> "$config"
-        echo "Version = 1" >> "$config"
-    fi
-    iniSet "RESAMPLE" "src-sinc-fastest"
+    iniSet "Version" "1" "" "[Audio-SDL]"
+    iniSet "RESAMPLE" "src-sinc-fastest" "" "[Audio-SDL]"
```

Comment:
I have done some more testing with the re-sample algorithm and I have found that "src-sinc-fastest" adds an unacceptable amount of audio latency. Setting the re-sample to "trivial" seems to be a much better solution as it gets rid of the audio crackle issue without adding latency.